Context.sublime-menu

```diff
@@ -5,35 +5,35 @@
         [
             {
                 "command": "lsp_symbol_references",
-                "caption": "Find Symbol References"
+                "caption": "Goto Reference…"
             },
             {
                 "command": "lsp_symbol_definition",
-                "caption": "Go To Symbol Definition"
+                "caption": "Goto Definition…"
             },
             {
                 "command": "lsp_symbol_type_definition",
-                "caption": "Go To Symbol Type Definition"
+                "caption": "Goto Type Definition…"
             },
             {
                 "command": "lsp_symbol_declaration",
-                "caption": "Go To Symbol Declaration"
+                "caption": "Goto Declaration…"
             },
             {
                 "command": "lsp_symbol_implementation",
-                "caption": "Go To Symbol Implementation"
+                "caption": "Goto Implementation…"
             },
             {
                 "command": "lsp_symbol_rename",
-                "caption": "Rename Symbol"
+                "caption": "Rename…"
             },
             {
                 "command": "lsp_code_actions",
-                "caption": "Code Actions"
+                "caption": "Code Actions…"
             },
             {
                 "command": "lsp_format_document",
-                "caption": "Format Document"
+                "caption": "Format View"
```

Comment:
I would think that `Format Document` is more user friendly.

A `View` is a ST implementation detail. Users probably don't know what a `View` is, and that is the reason why I don't think `Format View` is suitable. Imagine if it was called `Format Buffer` ). 

`Format File` is a bit shorter,  
but again, there is nothing wrong with `Format Document` either :)

Reply:
I went with Format File. The word "Document" is not ST-native. You and I know what it means, but an end-user who is not familiar with VSCode or the LSP spec doesn't know what it means.

Reply:
I've noticed this change (`Format file`) while using LSP and it struck me as odd because it's not formatting the file physically but just the buffer/document/view.